src/views/single-event/back-link.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Are this file being used?

Reply:
Same question for the rest of files after this one.

Reply:
Yeah loading from `single-event-blocks`